setup.py

```diff
@@ -136,6 +139,18 @@
 ENABLE_DOCUMENTATION_BUILD = os.environ.get(
     'GRPC_PYTHON_ENABLE_DOCUMENTATION_BUILD', False)
 
+def check_linker_need_libatomic():
+  """Test if linker on system needs libatomic.
```

Comment:
Nit: This docstring can all be on a single line.

Reply:
Done.